### DISCUSSION TITLE
Using Sceneform with ARCore Cloud Anchors

### DISCUSSION THREAD

**Discussion by tony-covais** [sceneview/sceneform-android#341] [Q&A]
Hello, I'm fairly new to the development game. I'm working on a project aiming to use Cloud Anchors to host objects which other users can reveal. I'm developing in android studio java. A lot of tutorials I've been using seem to be deprecated. Does anyone have any helpful suggestions for very basic hosting using Cloud Anchors and Sceneform?

Thanks.

**Comment by grassydragon** [comment]
Hi!
You can follow the documentation for Java (https://developers.google.com/ar/develop/cloud-anchors) and take a look at the sample ARCore application. After the Cloud Anchor is resolved you can create an `AnchorNode` using it.
